CHANGELOG.md

```diff
@@ -25,6 +25,20 @@ Please update your configuration accordingly.
 If you don't update the `actor-factory` value, you'll get a deprecation warning in the logs, and Cromwell will default back to
 **PAPI V1**
 
+**Requester Pays**
+
+Cromwell now supports [Requester Pays](https://cloud.google.com/storage/docs/requester-pays) feature for Cloud Storage. With Requester Pays enabled on your bucket,
+you can require requesters to include a billing project in their requests, thus billing the requester's project. If the bucket you are accessing has requester pays enabled,
+to access the file inside that bucket you need to mention the Google project id which can be billed for that request. The billing project information can be added in below ways. Cromwell will
+look for them in decreasing order of priorities.
+1. It can be added as `'google_project':'project-id'` as part of workflow options during workflow submission
+2. It can be included inside configuration file as shown in [`Getting started on Google Pipelines API`](http://cromwell.readthedocs.io/en/develop/tutorials/PipelinesApi101/) where you need to replace the `<google-billing-project-id>` with the project id **(HIGHLY RECOMMENDED)**
+3. If it is not mentioned using above 2 ways, Cromwell will use the default project that has been configured with gcloud
+
+It is highly recommended to add the project id as part of configuration if you don't want to specify as part of workflow options each time you submit request. The above ways are in
+descending order of priority, overriding the one below it if mentioned. So if you mention the project id using all three, Cromwell will use the one specified along with workflow options.
+Although, you just need to mention the project id using one way.
```

Comment:
I'd probably delete all sentences in this paragraph except the first since this was already said above.

Reply:
Thank you for these suggestions. I mean it! It was hard framing what actually needs to be done for requester pays (especially the order in which Cromwell picks up project-id), and all these changes makes it better! Thank you :)